app/models/participant_declaration/npq.rb

```diff
@@ -11,6 +11,7 @@ class ParticipantDeclaration::NPQ < ParticipantDeclaration
     "npq-executive-leadership" => "NPQEL",
     "npq-early-years-leadership" => "NPQEYL",
     "npq-leading-primary-mathematics" => "NPQLPM",
+    "npq-senco" => "NPQS",
```

Comment:
this needs to be agreed with the TRA team as we send outcomes to them and they need to set those identifiers on their side, let's discuss tomorrow